source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,167 +1,194 @@
 .. _mixin:
 
-Create and add custom bundles to your upstream Clear Linux system
-#################################################################
-
-|CL-ATTR| offers many curated bundles that you can install on your system to
-create your desired capabilities. If the available upstream bundles do not
-meet your needs, you can create and add your own custom bundles to your
-system using one of two methods. Note: Upstream refers to the official
-version of |CL|.
-
-The first method is to use the :ref:`mixer tool<mixer>` to create your own
-|CL| image and add your bundles to it.  Mixing your own |CL| image can
-give you great control and flexibility; however, you must act as an
-:abbr:`OSV (Operating System Vendor)` and maintain your releases and
-updates because you have forked from upstream.
-
-The second method is to use the :command:`mixin` tool, which also
-makes use of mixer to create custom bundles that you can add to your
-upstream |CL| system.  This  simpler method provides a “light” forking from
-upstream, which means you can continue to get upstream bundles and updates.
-If needed, you can easily revert your system back to the upstream version.
-
-This guide shows you how to accomplish the second method by following these
-steps:
-
-#. Set up the workspace.
-#. Copy your custom RPM package to the workspace.
-#. Create a bundle with your custom RPM package.
-#. Migrate your |CL| system to your custom mix.
-#. Add your custom bundle to your system.
-#. Optional: Revert your system back to 100% upstream.
-
-Set up the workspace
-********************
-
-#. Install the mixer bundle to enable mixer.
+mixin
+#####
 
-   .. code-block:: console
+The mixin tool is a custom content manager provided in the |CL-ATTR| that allows
```

Comment:
I would not use "custom content manager" because it is not a manager.  It's a tool.